newsService/src/main/java/com/kovan/app/service/NewsService.java

```diff
@@ -63,25 +72,57 @@ public NewsDto getTopHeadlines() {
 
         } while (pageSize <= newsDto.getTotalResults());
 
-        LocalDate dateOnly = LocalDate.parse(
-                articles.getFirst().getPublishedAt(),
-                DateTimeFormatter.ISO_DATE_TIME
-        );
-
-        if (!isNewsAlreadyInDb(dateOnly.toString())) {
-            NewsDto finalDto = NewsDto.builder()
-                    .articles(articles)
-                    .totalResults(articles.size())
-                    .publishedAt(dateOnly.toString())
-                    .build();
-
-            newsDto = service.saveNewsInDb(finalDto);
+        List<NewsDto.Article> newArticles = null;
```

Comment:
what is this name newArticles , latestArticles change to newsArticles, futureArticles

Reply:
changed to newsArticles and futureArticles